packages/nimble-components/src/select/index.ts

```diff
@@ -644,7 +644,11 @@ export class Select extends FormAssociatedSelect implements ErrorPattern {
     }
 
     protected override focusAndScrollOptionIntoView(): void {
-        super.focusAndScrollOptionIntoView();
+        const optionToFocus = this.firstSelectedOption;
```

Comment:
A few questions with this implementation:
- Any idea why calling `requestAnimationFrame` would break the drawer's animation? That seems like a browser bug unless there is something weird in the way FAST's DOM class is interacting with the browser's animations
- What about anything else that could cause `requestAnimationFrame`? Would anything calling `requestAnimationFrame` during the drawer's animation cause the animation to break? At the least, there is a call to `requestAnimationFrame` on line 653 below, but it seems unrealistic to avoid every possible call to `requestAnimationFrame` that could ever happen during a drawer's animation.

Reply:
There probably is a very specific browser bug. I don't see this in the Playwright WebKit browser, and it only happens if `scrollIntoView` is called in `requestAnimationFrame` (though I've only tried a couple other functions), and it only happens if the drawer is sliding in from the right. I can't really guess at the reason/mechanism for what we see happening. I can try to make a simplified reproducing case to file a browser bug with, but I don't think I should spend too much more time on this.

Reply:
I see. It wasn't clear to me from the description in the PR that the issue was the call to `scrollIntoView` within the `requestAnimationFrame` rather than the `requestAnimationFrame` call itself. Perhaps you could clarify that a little in the description.

This still seems like an odd fix for a drawer animation bug, but I agree that I'm not sure how much time we should spend on this since that combination of calls is probably unlikely to occur often.

Reply:
Got a [reproducing Stackblitz](https://stackblitz.com/edit/stackblitz-starters-ydmp5r?file=index.html,styles.css,script.js). It is broken on both Chromium and Firefox (but in different ways). Not sure about Safari. I'm going to file bugs.